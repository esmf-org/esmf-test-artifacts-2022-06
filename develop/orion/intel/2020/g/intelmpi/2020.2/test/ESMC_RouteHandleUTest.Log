build time -- 2022-03-12 04:16:11
20220312 064150.359 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220312 064150.360 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220312 064150.360 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220312 064150.360 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220312 064150.360 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220312 064150.360 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220312 064150.360 INFO             PET0 Running with ESMF Version   : v8.3.0b09
20220312 064150.361 INFO             PET0 ESMF library build date/time: "Mar 12 2022" "04:14:19"
20220312 064150.361 INFO             PET0 ESMF library build location : /work/noaa/da/mpotts/sandbox/intel_2020_intelmpi_g_develop
20220312 064150.361 INFO             PET0 ESMF_COMM                   : intelmpi
20220312 064150.364 INFO             PET0 ESMF_MOAB                   : enabled
20220312 064150.364 INFO             PET0 ESMF_LAPACK                 : enabled
20220312 064150.364 INFO             PET0 ESMF_NETCDF                 : enabled
20220312 064150.364 INFO             PET0 ESMF_PNETCDF                : disabled
20220312 064150.364 INFO             PET0 ESMF_PIO                    : enabled
20220312 064150.364 INFO             PET0 ESMF_YAMLCPP                : enabled
20220312 064150.367 INFO             PET0 Beginning Test, file ESMC_RouteHandleUTest.C, line 41
20220312 064150.368 INFO             PET0 Beginning Test, file ESMC_RouteHandleUTest.C, line 41NUMBER_OF_PROCESSORS 4
20220312 064203.547 INFO             PET0 PASS ESMC_RouteHandleWrite test, ESMC_RouteHandleUTest.C, line 269
20220312 064203.866 INFO             PET0 PASS ESMC_RouteHandleCreateFromFile test, ESMC_RouteHandleUTest.C, line 277
20220312 064203.868 INFO             PET0 PASS ESMC_FieldRegrid validation of RouteHandleWrite, ESMC_RouteHandleUTest.C, line 299
20220312 064203.868 INFO             PET0 PASS ESMC_FieldRegrid validation of RouteHandleCreateFromFile, ESMC_RouteHandleUTest.C, line 321
20220312 064203.869 INFO             PET0 Ending Test, file ESMC_RouteHandleUTest.C, line 345
20220312 064203.869 INFO             PET0 Finalizing ESMF
20220312 064150.362 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220312 064150.362 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220312 064150.362 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220312 064150.362 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220312 064150.362 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220312 064150.362 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220312 064150.362 INFO             PET1 Running with ESMF Version   : v8.3.0b09
20220312 064150.362 INFO             PET1 ESMF library build date/time: "Mar 12 2022" "04:14:19"
20220312 064150.362 INFO             PET1 ESMF library build location : /work/noaa/da/mpotts/sandbox/intel_2020_intelmpi_g_develop
20220312 064150.362 INFO             PET1 ESMF_COMM                   : intelmpi
20220312 064150.365 INFO             PET1 ESMF_MOAB                   : enabled
20220312 064150.365 INFO             PET1 ESMF_LAPACK                 : enabled
20220312 064150.365 INFO             PET1 ESMF_NETCDF                 : enabled
20220312 064150.365 INFO             PET1 ESMF_PNETCDF                : disabled
20220312 064150.365 INFO             PET1 ESMF_PIO                    : enabled
20220312 064150.365 INFO             PET1 ESMF_YAMLCPP                : enabled
20220312 064150.368 INFO             PET1 Beginning Test, file ESMC_RouteHandleUTest.C, line 41
20220312 064150.368 INFO             PET1 Beginning Test, file ESMC_RouteHandleUTest.C, line 41NUMBER_OF_PROCESSORS 4
20220312 064203.547 INFO             PET1 PASS ESMC_RouteHandleWrite test, ESMC_RouteHandleUTest.C, line 269
20220312 064203.866 INFO             PET1 PASS ESMC_RouteHandleCreateFromFile test, ESMC_RouteHandleUTest.C, line 277
20220312 064203.868 INFO             PET1 PASS ESMC_FieldRegrid validation of RouteHandleWrite, ESMC_RouteHandleUTest.C, line 299
20220312 064203.868 INFO             PET1 PASS ESMC_FieldRegrid validation of RouteHandleCreateFromFile, ESMC_RouteHandleUTest.C, line 321
20220312 064203.869 INFO             PET1 Ending Test, file ESMC_RouteHandleUTest.C, line 345
20220312 064203.869 INFO             PET1 Finalizing ESMF
20220312 064150.359 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220312 064150.360 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220312 064150.360 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220312 064150.360 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220312 064150.360 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220312 064150.360 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220312 064150.360 INFO             PET2 Running with ESMF Version   : v8.3.0b09
20220312 064150.361 INFO             PET2 ESMF library build date/time: "Mar 12 2022" "04:14:19"
20220312 064150.361 INFO             PET2 ESMF library build location : /work/noaa/da/mpotts/sandbox/intel_2020_intelmpi_g_develop
20220312 064150.361 INFO             PET2 ESMF_COMM                   : intelmpi
20220312 064150.364 INFO             PET2 ESMF_MOAB                   : enabled
20220312 064150.364 INFO             PET2 ESMF_LAPACK                 : enabled
20220312 064150.364 INFO             PET2 ESMF_NETCDF                 : enabled
20220312 064150.364 INFO             PET2 ESMF_PNETCDF                : disabled
20220312 064150.364 INFO             PET2 ESMF_PIO                    : enabled
20220312 064150.365 INFO             PET2 ESMF_YAMLCPP                : enabled
20220312 064150.368 INFO             PET2 Beginning Test, file ESMC_RouteHandleUTest.C, line 41
20220312 064150.368 INFO             PET2 Beginning Test, file ESMC_RouteHandleUTest.C, line 41NUMBER_OF_PROCESSORS 4
20220312 064203.561 INFO             PET2 PASS ESMC_RouteHandleWrite test, ESMC_RouteHandleUTest.C, line 269
20220312 064203.866 INFO             PET2 PASS ESMC_RouteHandleCreateFromFile test, ESMC_RouteHandleUTest.C, line 277
20220312 064203.880 INFO             PET2 PASS ESMC_FieldRegrid validation of RouteHandleWrite, ESMC_RouteHandleUTest.C, line 299
20220312 064203.881 INFO             PET2 PASS ESMC_FieldRegrid validation of RouteHandleCreateFromFile, ESMC_RouteHandleUTest.C, line 321
20220312 064203.881 INFO             PET2 Ending Test, file ESMC_RouteHandleUTest.C, line 345
20220312 064203.881 INFO             PET2 Finalizing ESMF
20220312 064150.363 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220312 064150.364 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220312 064150.364 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220312 064150.364 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220312 064150.364 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220312 064150.364 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220312 064150.364 INFO             PET3 Running with ESMF Version   : v8.3.0b09
20220312 064150.364 INFO             PET3 ESMF library build date/time: "Mar 12 2022" "04:14:19"
20220312 064150.364 INFO             PET3 ESMF library build location : /work/noaa/da/mpotts/sandbox/intel_2020_intelmpi_g_develop
20220312 064150.364 INFO             PET3 ESMF_COMM                   : intelmpi
20220312 064150.365 INFO             PET3 ESMF_MOAB                   : enabled
20220312 064150.365 INFO             PET3 ESMF_LAPACK                 : enabled
20220312 064150.365 INFO             PET3 ESMF_NETCDF                 : enabled
20220312 064150.365 INFO             PET3 ESMF_PNETCDF                : disabled
20220312 064150.365 INFO             PET3 ESMF_PIO                    : enabled
20220312 064150.365 INFO             PET3 ESMF_YAMLCPP                : enabled
20220312 064150.368 INFO             PET3 Beginning Test, file ESMC_RouteHandleUTest.C, line 41
20220312 064150.368 INFO             PET3 Beginning Test, file ESMC_RouteHandleUTest.C, line 41NUMBER_OF_PROCESSORS 4
20220312 064203.560 INFO             PET3 PASS ESMC_RouteHandleWrite test, ESMC_RouteHandleUTest.C, line 269
20220312 064203.866 INFO             PET3 PASS ESMC_RouteHandleCreateFromFile test, ESMC_RouteHandleUTest.C, line 277
20220312 064203.881 INFO             PET3 PASS ESMC_FieldRegrid validation of RouteHandleWrite, ESMC_RouteHandleUTest.C, line 299
20220312 064203.882 INFO             PET3 PASS ESMC_FieldRegrid validation of RouteHandleCreateFromFile, ESMC_RouteHandleUTest.C, line 321
20220312 064203.882 INFO             PET3 Ending Test, file ESMC_RouteHandleUTest.C, line 345
20220312 064203.882 INFO             PET3 Finalizing ESMF
