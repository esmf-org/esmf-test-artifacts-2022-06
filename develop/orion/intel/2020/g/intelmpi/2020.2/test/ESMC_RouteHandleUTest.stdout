build time -- 2022-03-19 03:29:06
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
Beginning Test, file ESMC_RouteHandleUTest.C, line 41
Beginning Test, file ESMC_RouteHandleUTest.C, line 41
Beginning Test, file ESMC_RouteHandleUTest.C, line 41NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_RouteHandleUTest.C, line 41NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_RouteHandleUTest.C, line 41
Beginning Test, file ESMC_RouteHandleUTest.C, line 41NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_RouteHandleUTest.C, line 41
Beginning Test, file ESMC_RouteHandleUTest.C, line 41NUMBER_OF_PROCESSORS 4
PASS ESMC_RouteHandleWrite test, ESMC_RouteHandleUTest.C, line 269
PASS ESMC_RouteHandleWrite test, ESMC_RouteHandleUTest.C, line 269
PASS ESMC_RouteHandleWrite test, ESMC_RouteHandleUTest.C, line 269
PASS ESMC_RouteHandleWrite test, ESMC_RouteHandleUTest.C, line 269
PASS ESMC_RouteHandleCreateFromFile test, ESMC_RouteHandleUTest.C, line 277
PASS ESMC_RouteHandleCreateFromFile test, ESMC_RouteHandleUTest.C, line 277
PASS ESMC_FieldRegrid validation of RouteHandleWrite, ESMC_RouteHandleUTest.C, line 299
PASS ESMC_FieldRegrid validation of RouteHandleWrite, ESMC_RouteHandleUTest.C, line 299
PASS ESMC_FieldRegrid validation of RouteHandleCreateFromFile, ESMC_RouteHandleUTest.C, line 321
PASS ESMC_FieldRegrid validation of RouteHandleCreateFromFile, ESMC_RouteHandleUTest.C, line 321
Ending Test, file ESMC_RouteHandleUTest.C, line 345
Ending Test, file ESMC_RouteHandleUTest.C, line 345
PASS ESMC_RouteHandleCreateFromFile test, ESMC_RouteHandleUTest.C, line 277
PASS ESMC_FieldRegrid validation of RouteHandleWrite, ESMC_RouteHandleUTest.C, line 299
PASS ESMC_FieldRegrid validation of RouteHandleCreateFromFile, ESMC_RouteHandleUTest.C, line 321
Ending Test, file ESMC_RouteHandleUTest.C, line 345
PASS ESMC_RouteHandleCreateFromFile test, ESMC_RouteHandleUTest.C, line 277
PASS ESMC_FieldRegrid validation of RouteHandleWrite, ESMC_RouteHandleUTest.C, line 299
PASS ESMC_FieldRegrid validation of RouteHandleCreateFromFile, ESMC_RouteHandleUTest.C, line 321
Ending Test, file ESMC_RouteHandleUTest.C, line 345
Ending Test, file ESMC_RouteHandleUTest.C, line 345 PET 1 Test Elapsed Time 368.485 msec.
Ending Test, file ESMC_RouteHandleUTest.C, line 345 PET 2 Test Elapsed Time 160.035 msec.
Ending Test, file ESMC_RouteHandleUTest.C, line 345 PET 0 Test Elapsed Time 507.797 msec.
Ending Test, file ESMC_RouteHandleUTest.C, line 345 PET 3 Test Elapsed Time 507.966 msec.
